src/components/Personalization/index.js

```diff
@@ -84,7 +84,7 @@ const createPersonalization = ({ config, logger, eventManager }) => {
 
         showContainers();
       },
-      onResponseError() {
+      onRequestFailure() {
         showContainers();
```

Comment:
So is this the first use case of `response.getErrorsByCode`? Because if the status code is considered successful but Personalization errors out, those containers will stay hidden.

Reply:
I don't see a case here where containers would stay hidden, but please point it out if I'm missing something. If the request itself fails, `onRequestFailure` is called, and containers will be shown. If the request succeeds (Konductor returns a valid response), then `onResponse` will get called, [`response.getPayloadsByType(DECISIONS_HANDLE);`](https://github.com/adobe/alloy/pull/283/files#diff-74183009f12144cdb4dccd827ce535d5L81) will return 0 fragments, and then containers will be shown.

If we wanted to detect that a personalization error occurred and do something specific about it, that would be an option and that's where we would use `response.getErrorsByCode`, but I don't know what specific action we would take at that point.